crates/jstz_node/Cargo.toml

```diff
@@ -24,7 +24,7 @@ parking_lot = "0.12.1"
 reqwest = { version = "0.11.22", features = ["json"] }
 serde = { version = "1.0.183", features = ["derive"] }
 thiserror = "1.0.56"
-tokio = { version = "1.33.0", features = ["fs"] }
+tokio = { version = "1.33.0", features = ["fs", "rt-multi-thread"] }
```

Comment:
what does this do?

Reply:
Tokio seemed to give me a compiler error without this feature flag